lib/portage/util/digraph.py

```diff
@@ -76,6 +83,9 @@ def remove(self, node):
     def update(self, other):
         """
         Add all nodes and edges from another digraph instance.
+
+        other: another digraph instance,
+        can't be type annotated according to PEP 484 due to cyclic imports
```

Comment:
Is there really no way around that?

Reply:
Only recently discovered "forward reference" type hints. Fixed.